app/controllers/api/base_controller.rb

```diff
@@ -118,5 +119,10 @@ def api_error(type, error)
       render :json => ErrorSerializer.new(type, error).serialize, :status => Rack::Utils.status_code(type)
       log_api_response
     end
+
+    def ensure_pagination
+      params["limit"] ||= Settings.api.max_results_per_page
```

Comment:
Should a user be allowed to pass a value *higher* that the max_results_per_page?  That is, if they pass `:limit => 1000000`, it could bring down the server, which seems like a possible DoS.